app/utils/index.ts

```diff
@@ -0,0 +1,18 @@
+class Generator {
+  private *idGenerator(start: number, end: number, step: number) {
+    for (let i = start; i < end; i += step) {
+      yield i;
+    }
+  }
+  private newGen = this.idGenerator(this.start, this.end, this.step);
```

Comment:
Good to separate blocks of code with an empty line. A good reason to have linting/formating setup.